cmdx.py

```diff
@@ -1522,13 +1524,13 @@ def __or__(self, other):
             >>> _new()
             >>> parent = createNode("transform", "parent")
             >>> child =  createNode("transform", "child", parent)
-            >>> parent | "child"
-            |parent|child
+            >>> (parent | "child").path()
+            '|parent|child'
 
             # Stackable too
             >>> grand =  createNode("transform", "grand", child)
-            >>> parent | "child" | "grand"
-            |parent|child|grand
+            >>> (parent | "child" | "grand").path()
+            '|parent|child|grand'
```

Comment:
BY the way, it might be a bit hacky, but we could make this more cmdx friendly by adding something like the following to `__or__`:
```python
if not isinstance(other, string_types):
    if isinstance(other, DagNode):
        other = other.name()
```
Then, we could provide DagNodes and this would work:
```
>>> parent | child | grand
<cmdx.DagNode : '|parent|child|grand'>

>>> (parent | child | grand).path()
'|parent|child|grand'
```

Reply:
Huh? Wouldn't `grand` then already be a `cmdx.DagNode` at which point.. there would be no need to use `|`? 🤔 Maybe if you put together a complete example would it be easier to see what you've got in mind.

Reply:
> Wouldn't grand then already be a cmdx.DagNode at which point.. there would be no need to use |? 

You could still use the string version of it, in combination with DagNodes:
```python
>>> parent = cmdx.createNode("transform", "parent")
>>> child =  cmdx.createNode("transform", "child", parent)
>>> grand =  cmdx.createNode("transform", "grand", child)
>>> parent | "child" | grand
<cmdx.DagNode : '|parent|child|grand'>
```

Although, having a second thought on this, it is probably unnecessary. I thought it could be nice to have some flexibility, but it perhaps would just add confusion.

I guess the power of using the strings here is to be able to do things like:
```python
>>> parent | "chi*"
<cmdx.DagNode : '|parent|child'>
>>> parent | "chi*" | "*"   
<cmdx.DagNode : '|parent|child|grand'>
```